src/views/connected/structure/popins/popinSelectionCoordinateur.js

```diff
@@ -94,10 +94,10 @@ function popinSelectionCoordinateur({
   );
 }
 
-popinSelectionCoordinateur.propTypes = {
+PopinSelectionCoordinateur.propTypes = {
   setOpenModal: PropTypes.func.isRequired,
   conseillersActifs: PropTypes.array.isRequired,
   structure: PropTypes.object.isRequired,
 };
 
-export default popinSelectionCoordinateur;
+export default PopinSelectionCoordinateur;
```

Comment:
bizarre mais non pris en compte pour le name du file :/ 
![image](https://github.com/anct-cnum/dashboard/assets/68587983/9cc976d4-a9b0-481d-9ab1-894ecfbfff6c)

Reply:
question de config : dans git config core.ignorecase doit être à false au lieu de true sut ta machine
https://linuxhint.com/commit-case-sensitive-only-filenames-changes-in-git/